noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1168,10 +1605,15 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST12">
         <title>Records within leaf classes</title>
+        <description>
+            Provides information about the number of Records per Series and Class
```

Comment:
`Provides information about the number of records per series and class, and tests whether any record belongs to a class which has other classes as children (i.e. the record has a class sibling)`